### PR TITLE
set up testing and completed step 3

### PR DESCRIPTION
**Pull request by ggr1992**

### CODE REVIEW

File: __tests__/get-api-categories.test.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+describe('Testing the /api/categories', () => {
+    test('testing that get api/categories returns the right information and returns status 200', () => {
+        return request(app)
+        .get("/api/categories")
+        .expect(200)
+            .then((result) => {
+                const categories = result.body.categories
+                expect(categories.length).toBe(4)
+                
+                expect(categories[0]).toHaveProperty('slug')
```
Comment (Beth-McCormack) [inline __tests__/get-api-categories.test.js]: This is only checking the first item in the array. Can you change this to check each item in the array looks how we expect?

File: app.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+const express = require("express");
+const { manageCategories } = require("./controllers/categories.controllers");
+
+const app = express()
+
+app.use(express.json())
```
Comment (Beth-McCormack) [inline app.js:6]: Please remove this until it is needed

**Review by Beth-McCormack — COMMENTED** [review]
Nice work! Please see my comments and submit another PR when done :)

**Review by leekli — APPROVED** [review]
Thanks for making the change Glen, you are free to merge this branch :)

**Pull request merged**